src/machines/child_workflow_state_machine.rs

```diff
@@ -62,13 +163,23 @@ pub(super) struct StartEventRecorded {}
 impl StartEventRecorded {
     pub(super) fn on_child_workflow_execution_started(
         self,
+        attrs: ChildWorkflowExecutionStartedEventAttributes,
     ) -> ChildWorkflowMachineTransition<Started> {
-        unimplemented!()
+        ChildWorkflowMachineTransition::ok(vec![
+            ChildWorkflowCommand::Start(attrs.workflow_execution.expect("ChildWorkflowExecutionStartedEventAttributes.workflow_execution should not be empty"))
```

Comment:
Rather than `expect`ing here, the conversion from the history event should package exactly what is needed into the machine event. Only some subset of the attributes are required here, in any case.

Reply:
Same for the other places this happens